BEIMA.Client/src/shared/Notifications/Notification.js

```diff
@@ -39,7 +39,7 @@ export async function success(title, message){
 export async function warning(title, message){
   return await notification.fire({
         title: title,
-        html: message.join(''),
+        html: message + '',
```

Comment:
The join part was to remove commas from the warning. In the case where I passed it an array of messages to display when I was testing it on the Add Device page for empty fields I got a list like: 
  field1 is blank
, field2 is blank
, ...
, field10 is blank

after adding the .join it removed the commas so it was just 1 item per line:
field1 is blank
field2 is blank
...
field10 is blank

I think it says join isn't a function because a single string was passed in. It probably needs a conditional to determine if join needs to run, or, another type added for WarnMultipe that has it and convert "warning" to look like the success/error ones that don't have the "html:" property set.

Reply:
added #356 and assigned it to myself to extend Notifications.js with a function for multiple warnings, convert the existing warning function to work like success/error and update this page once completed.